### DISCUSSION TITLE
Can't start MySQL server: Bind on TCP/IP port: Address already in use

### DISCUSSION THREAD

**Discussion by skerit** [nektos/act#2362] [Q&A]
I'm trying to start mysql, using the `catthehacker/ubuntu:full-latest` image.

My run command looks like this:

```
          sudo usermod -d /var/lib/mysql/ mysql
          sudo /etc/init.d/mysql start
```

But I get the following error:

```
2024-06-12T07:10:58.249050Z 0 [ERROR] [MY-010262] [Server] Can't start server: Bind on TCP/IP port: Address already in use
2024-06-12T07:10:58.249060Z 0 [ERROR] [MY-010257] [Server] Do you already have another mysqld server running on port: 3306 ?
```

I don't have another mysql server inside the container. I do have one in another container, and that port is exposed, but this container doesn't expose its ports so ... what's the problem here?

**Comment by ChristopherHX** [comment]
>  this container doesn't expose its ports

it does expose all ports, due to network mode host

If you don't want to expose them, act looks for jobs.job.container to be set.
